pkg/mobile/types.go

```diff
@@ -94,6 +94,9 @@ func (at AppTypes) String() string {
 //ValidAppTypes is a list of valid app types
 var ValidAppTypes = AppTypes{"cordova", "android", "iOS"}
 
+//ServiceTypes are the service types that we are aware of and support
+var ServiceTypes = []string{"fh-sync-server", "keycloak", "custom"}
```

Comment:
I'm uncomfortable with this in general - not specific to this PR; I don't like that we have to recompile, rebuild and redeploy the API server anytime we want to add a new mobile service.

Reply:
yes agree, however I am chatting with the service catalog guys around having labels on the secrets https://github.com/kubernetes-incubator/service-catalog/issues/1222 which I think will help. Also can be moved out to an env var at somepoint